basesqlmodel/main.py

```diff
@@ -69,7 +69,7 @@ async def get(
         cls: Type[Self],
         session: AsyncSession,
         *args: BinaryExpression,
-        load_strategy: Dict[str, LoadStrategy] = None,
+        load_strategy: Union[Dict[str, LoadStrategy], None] = None,
```

Comment:
```suggestion
        load_strategy: Dict[str, LoadStrategy] | None= None,
```

Reply:
> 

won't this fail on python versions < 3.10 ? That's the reason I used Union. I put the next lowest possible version in the pyproject.toml that will work with all existing dependencies and sqlmodel = "^0.0.8"

Reply:
The `__future__.annotations` make it pass.

Reply:
got it, thanks